runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -7641,4 +7777,27 @@ void am_debugPrep(amRequest_t* req) {
   }
 }
 
+
+static
```

Comment:
Does this need to be in a `CHPL_COMM_DEBUG` ifdef? (Wondering if we'll get warnings about it not being used if we don't)

Reply:
Yeah, it does. I tried taking out of that ifdef and we get an error about `op_uses_on_bundle()` being undefined in it, as well as about the `.seq` (sequence number) elements of the AM header being undefined. All of those are debug-only.

Reply:
Ah, I misread. I thought it wasn't in one, and was asking if it should be put in one, but looking at the code again I see it's part of a pretty large DEBUG block already. Nevermind me.